### DISCUSSION TITLE
Set Tailwind Config value from API callback.

### DISCUSSION THREAD

**Discussion by Huvinesh-Rajendran-12** [tailwindlabs/tailwindcss#10476] [Help]
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/81321926/216509480-8100cabf-4a62-47e4-b8ee-650807284c8e.png">

Hi, I am trying to set the value in the Tailwind config file from an API request callback, but I'm keep on getting this error which states the getFontConfig is not a function. Below is the code of the API callback. 

<img width="1088" alt="image" src="https://user-images.githubusercontent.com/81321926/216509984-201e40aa-39bf-4647-b47a-77bb72b46e16.png">

**Comment by dasdware** [comment]
In your fontAPI, you're calling the getFontConfig function when exporting it. Therefore, you export whatever the function returns, not the function itself.
